java-to-zod-core/src/main/java/sh/ivan/zod/plugins/PluginParameters.java

```diff
@@ -0,0 +1,320 @@
+package sh.ivan.zod.plugins;
+
+import cz.habarta.typescript.generator.*;
+import java.io.Serializable;
+import java.util.Collections;
+import java.util.List;
+import lombok.Setter;
+import org.apache.maven.plugins.annotations.Parameter;
+import org.gradle.api.tasks.Input;
+import org.gradle.api.tasks.Optional;
+
+@Setter
+public class PluginParameters implements Serializable {
+    private static final long serialVersionUID = 1L;
+
+    /**
+     * Prefix added to generated schema names.
```

Comment:
Do these comments get added to any kind of Gradle online documentation system? Would love to retain these somehow. Worst case, we can always link to the generated Maven configuration docs. Here's what that looks like: https://java-to-zod.ivan.sh/java-to-zod-maven-plugin/generate-mojo.html

This reminds me, we may want to find a way to publish any generated Gradle documentation to the site along with the Maven version. Site: https://java-to-zod.ivan.sh/